app/code/core/Mage/Catalog/Model/Layer/Filter/Decimal.php

```diff
@@ -165,15 +165,15 @@ public function getMinValue()
      */
     public function getRange()
     {
-        $range = $this->getData('range');
+        $range = (float)$this->getData('range');
         if (!$range) {
             $maxValue = $this->getMaxValue();
             $index = 1;
-            do {
+            while (!isset($items) || $range > self::MIN_RANGE_POWER && count($items) < 2) {
```

Comment:
changing this to a `while` from a `do...while` makes it harder to understand. Is this rule of the coding standard important enough so we have to follow it?

Reply:
No, Thats can be closed. Already opened issue for that.